qm-status: add page (#9180)
tldr-pages/tldr: pages/linux/qm-status.md

# qm status

> Show virtual machine status.
> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.

> - Show the status of the given virtual machine:

Reviewer: ```suggestion
- Show the status of a specific virtual machine:
```

> - Show the status of a specific virtual machine:

Reviewer 2: I think display is better verb
```suggestion
- Display the status of a specific virtual machine:
```

`qm status {{vm_id}}`

> - Show verbose output format:

Reviewer: ```suggestion
- Show verbose output of a specific virtual machine:
```
I think this would be better.
